### ISSUE TITLE
Features

### ISSUE THREAD

**Issue by pacmanw2** [darcikhey27/dustinwebsite#2]
Define a list of features for the website. Something like the template below should help.

# List of Pages

# General Functional Requirements

# Page Specific Functionality

**Comment by darcikhey27** [comment]
I think 2 pages should be enough for this simple website.

# list of pages,
* home page, 
* services page,
** Services page should display a calendar where customer can fill out a form to book an appointment

General functionality for the whole website,
maintain a list of customers in a database,
Have an admin page that shows bookings, customer info etc...

**Comment by pacmanw2** [comment]
Django has an admin page by default.

We can use a MySQL database for bookings.

**Comment by pacmanw2** [comment]
Does the admin want CRUD functionality with customers? Like, create a customer or update their info.

**Comment by darcikhey27** [comment]
Yeah, I think it would be good to have CRUD functionality that way if a customer cancels etc.. it can be removed.

**Comment by pacmanw2** [comment]
Have you thought about hosting options? I've been looking at some and [DreamHost](https://www.dreamhost.com/hosting/shared/) looks pretty alright. It supports Python.

**Comment by darcikhey27** [comment]
I haven't thought much about hosting, anything that is cheap would be fine. When you buy hosting does it come with a domain/email etc... I need to refresh my mind about how hosting works nowadays.

**Comment by pacmanw2** [comment]
Some of then come with a domain & email. The cheapest option from DreamHost only comes with the domain.  The email costs a little more to add.
